scripts/app.js

```diff
@@ -0,0 +1,82 @@
+console.log("he");
+// Input and output contents declared
+const chatbotIO = [
+    {
+        input : "hello",
+        output : ["Hello", "Hi", "Hey"]
+    },
+    {
+        input : "hi",
+        output : ["Hello", "Hi", "Hey"]
+    },
+    {
+        input : "hey",
+        output : ["Hello", "Hi", "Hey"]
+    },
+    {
+        input  : "what is your favourite color?",
+        output : ["I am not sure", "I like every color", "Orange"]
+    },
+    {
+        input : "how are you?",
+        output : ["Great", "Cool", "Good"]
+    }]
+
+    // just check chatbot input output content on console 
+    const list = chatbotIO.filter(io => {return io;});
+    console.log(list);
+
+    // random fuction, every time show rondom answer
+    const random = (io) => { return io[0].output[Math.floor(Math.random()*3)];}
```

Comment:
This function can be written in one line without the curly braces and return:
```javascript
const random = (io) => io[0].output[Math.floor(Math.random()*3)];
```

Some of your other one-line functions can be shortened as well.